src/main/resources/build.properties

```diff
@@ -6,3 +6,4 @@ springerNatureAPIKey=${springerNatureAPIKey}
 astrophysicsDataSystemAPIKey=${astrophysicsDataSystemAPIKey}
 ieeeAPIKey=${ieeeAPIKey}
 biodiversityHeritageApiKey=${biodiversityHeritageApiKey}
+Catalogs=Create property
```

Comment:
There was something going wrong here. This line needs to be removed, doesn't it?